device.go

```diff
@@ -0,0 +1,71 @@
+package qsc
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/byuoitav/connpool"
+)
+
+type DSP struct {
+	Address string
+	Pool    *connpool.Pool
+	Log     Logger
+}
```

Comment:
let's make these fields private since we have a `New()` that we want people to use